### PR TITLE
Bio-Formats 6.0.0-m3

### PR DESCRIPTION
**Pull request by sbesson**
Extend history to cover feature additions made between 6.0.0-m2 and 6.0.0-m3 milestones - see https://github.com/openmicroscopy/bioformats/milestone/64?closed=1

### CODE REVIEW

File: sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -37,6 +38,8 @@ New file formats:
 
 * KLB
    - added a new reader to support the reading of the Keller Lab Block (KLB) files
+* CV7000
+   - added a new reader to support the reading of Yokogawa CV7000 datasets
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: Not sure we need these "to support the reading of", surely "for" suffices?

File: sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -56,19 +64,36 @@ File format fixes and improvements:
    - added support for reading OME-TIFF with pyramidal resolutions stored as
      SubIFDs
    - added support for writing OME-TIFF with pyramidal resolutions
+   - add support for companion OME-TIFF filesets where TIFF does not link back
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: "added" above, not "add"

File: sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -46,6 +49,11 @@ File format fixes and improvements:
    - improved file grouping and file-to-series mapping for multi-file datasets
 * Image Pro
    - added support for Image Pro Plus .ips set
+* Leica SCN
+   - improve support for Vera datasets
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: should be ``improved`` to match the rest

File: sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -46,6 +49,11 @@ File format fixes and improvements:
...
+* Micro-Manager
+   - improve handling of very large :file:`*_metadata.txt` files
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: same

File: sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -46,6 +49,11 @@ File format fixes and improvements:
...
+   - prevent ``NumberFormatException`` for invalid double values
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: same

File: sphinx/about/whats-new.rst [hunk 6]
```diff
@@ -56,19 +64,36 @@ File format fixes and improvements:
    - added support for reading OME-TIFF with pyramidal resolutions stored as
      SubIFDs
    - added support for writing OME-TIFF with pyramidal resolutions
+   - add support for companion OME-TIFF filesets where TIFF does not link back
+     to the metadata file
+* PerkinElmer Operetta
+   - improve support to handle datasets generated by the Harmony software
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: same

File: sphinx/about/whats-new.rst [hunk 7]
```diff
@@ -56,19 +64,36 @@ File format fixes and improvements:
...
 * updated the updater message in the Fiji plugin (thanks to Jan Eglinger)
-* added ``-no-flat`` option to the command-line tools to convert files with
-  sub-resolutions
+* Command-line tools
+
+  - added ``-no-flat`` option to the command-line tools to convert files with
+    sub-resolutions
+  - added ``-pyramid-scale`` and ``-pyramid-resolutions`` options to
+    generate sub-resolution during conversion
+  - remove ``Plate`` elements when ``-series`` is passed as an option
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: removed

File: sphinx/about/whats-new.rst [hunk 8]
```diff
@@ -56,19 +64,36 @@ File format fixes and improvements:
...
+  - extend usage to describe available formats, extensions and compressions
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: extended

File: sphinx/about/whats-new.rst [hunk 9]
```diff
@@ -56,19 +64,36 @@ File format fixes and improvements:
...
 
 * updated the updater message in the Fiji plugin (thanks to Jan Eglinger)
-* added ``-no-flat`` option to the command-line tools to convert files with
-  sub-resolutions
+* Command-line tools
+
+  - added ``-no-flat`` option to the command-line tools to convert files with
+    sub-resolutions
+  - added ``-pyramid-scale`` and ``-pyramid-resolutions`` options to
+    generate sub-resolution during conversion
```
Comment (jburel) [inline sphinx/about/whats-new.rst]: sub-resolutions?

**Comment by sbesson** [comment]
@jburel @mtbc  thanks. 4c03339 should address all the comments

File: sphinx/about/whats-new.rst [hunk 10]
```diff
@@ -36,7 +37,9 @@ Bio-Formats API changes:
 New file formats:
 
 * KLB
-   - added a new reader to support the reading of the Keller Lab Block (KLB) files
+   - added a new reader for the Keller Lab Block (KLB) files
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: maybe not "the"

Comment (sbesson) [reply sphinx/about/whats-new.rst]: :+1:

**Comment by mtbc** [comment]
Looking much better, :+1:

**Pull request merged**